src/components/Snackbar/internal/SnackbarContent/styled.ts

```diff
@@ -3,6 +3,7 @@ import styled from "styled-components";
 export const Container = styled.div<{
   boxShadow: string;
   background: string;
+  text?: string;
```

Comment:
```suggestion
  textColor?: string;
```

こんな感じかなー、パッと見だと色の情報ではなくコンテンツの文字列が入るように思えた。

Reply:
確かに 👀